labelbox/__init__.py

```diff
@@ -2,6 +2,10 @@
 __version__ = "3.36.1"
 
 from labelbox.client import Client
+from labelbox.data import annotation_types as types
```

Comment:
This is going to break potentially. There are additional dependencies required for this module.

Reply:
Since the models themselves and related utilities are tightly coupled, this only works if you import labelbox[data] to include the extra deps. If users only install labelbox, this import will fail.

Reply:
you could get around this with some sort of lazy loading of this module. Or you could have users do something like:
from labelbox.data import annotation_types as lb_types.

Reply:
ideally we standarize to lb_*, 
from labelbox import annotation_types as lb_types
from labelbox import metrics as lb_metrics

Reply:
I think a second import in the notebooks makes the dependency `labelbox[data]` more transparent. It would be nice to show this dependency in the import statement e.g.  `from labelbox.data import types as lb_types`. I'll update this.